build.gradle

```diff
@@ -141,6 +141,7 @@ subprojects {
     version = '5.2.3-SNAPSHOT'
     ext {
         bouncyCastleVersion = '1.70'
+        commonsLang3Version = '3.12.0'
```

Comment:
this does not seem important enough to add an extra deopendency

Reply:
@pjfanning done. please review again.